### PR TITLE
default shipping and billing

### PR DESCRIPTION
**Pull request by PhilippSpo**
#### Changes
- Change variable names `defaultShippingAddressId` and `defaultBillingAddressId` to `defaultShippingAddress` and `defaultBillingAddress` ([docs](http://dev.commercetools.com/http-api-projects-customers.html#customerdraft))
- Add intergration tests for setting default shipping and billing address (should have added it in the first place)
- Adapt unit test to new variable names

cc/ @hajoeichler @emmenko 

@ct-dali can you pls quickly review?

### CODE REVIEW

File: tests/integration/customer-import.js [hunk 1]
```diff
@@ -28,37 +28,38 @@ describe('customer import module', function () {
...
+        return client[service].process(({ body: { results } }) => {
+          return Promise.map(results, (customer) => {
+            return client[service].byId(customer.id)
+            .delete(customer.version)
+          })
         })
+      }
+      deleteAll('customers')
+      .then(() => {
+        return deleteAll('customerGroups')
```
Comment (ghost) [inline tests/integration/customer-import.js]: you can do without the braces and `return`.

File: tests/integration/customer-import.js [hunk 2]
```diff
@@ -134,22 +135,109 @@ describe('customer import module', function () {
...
+  (done) => {
+
+    const customer = {
+      email: 'philipp.sporrer@commercetools.de',
+      addresses: [{
+        streetName: 'Ernst-Platz-Straße 45a',
+        postalCode: '80992',
+        city: 'München',
+        country: 'DE'
+      }]
```
Comment (ghost) [inline tests/integration/customer-import.js:165]: this is your actual address?? are you ok with this being public?

**Comment by ghost** [comment]
Looks good 👍

File: tests/integration/customer-import.js (hunk 2 above)
Comment (PhilippSpo) [inline tests/integration/customer-import.js:165]: it is actually. Didn't really second guess this. Thanks.

Comment (ghost) [inline tests/integration/customer-import.js:165]: in this case you should rewrite history of this repository so that this commit doesn't get stored.

**Pull request merged**